checks/__init__.py

```diff
@@ -350,12 +356,15 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         self.service_checks = []
         self.instances = instances or []
         self.warnings = []
+        self.check_version = None
         self.library_versions = None
         self.last_collection_time = defaultdict(int)
         self._instance_metadata = []
         self.svc_metadata = []
         self.historate_dict = {}
 
+        self.set_check_version(manifest=self._get_check_manifest())
```

Comment:
I don't think this would fail, if `load_manifest()` fails to be able to load the manifest, it will return `None`. I don't think the `inspect` calls or `os.path` calls will error out.